### DISCUSSION TITLE
Popover can't use fixed strategy

### DISCUSSION THREAD

**Discussion by IgorMCesar** [mantinedev/mantine#5662] [Feature requests]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

7.4.1

### What package has an issue?

@mantine/core

### What framework do you use?

Next.js

### In which browsers you can reproduce the issue?

None

### Describe the bug

At the moment the lib used to position the Popover dropdown is `@floating-ui/react`, and it provides an option for the positioning strategy. The file `use-popover.ts`, which Popover uses, doesn't allow for the strategy to be passed as an option to `useFloating`.  

In cases the `Popover.Target` is fixed or sticky, the absolute strategy will flicker `Popover.Dropdown`'s position during the scroll.

![image](https://github.com/mantinedev/mantine/assets/5064518/631de176-8e35-4e28-98a3-af431da5c841)

### Possible fix

Docs reference: https://floating-ui.com/docs/useFloating#strategy

File to be changed: https://github.com/mantinedev/mantine/blob/master/packages/%40mantine/core/src/components/Popover/use-popover.ts

### Self-service

- [ ] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment]
I will check if it is possible and add support for fixed strategy to popover in one of the next minor releases

**Reply by IgorMCesar** [reply]
@rtivital any news about this?

**Reply by rtivital** [reply]
It was added in 7.8

**Comment by thinkdj** [comment]
@rtivital To account for a11y, I was trying to use a popover to show related info. Is there a reason opening the popover onHover is not considered, like click+hover complications? I understand hovercards exist for this use case, but why limit an additional functionality of a popover?

**Reply by rtivital** [reply]
In your application you can use Popover any way you like, Mantine does not have any functionality limitations

**Reply by thinkdj** [reply]
@rtivital I meant, I didn't see a direct non-controlled way to open the popover on hover. It was for click only.

**Reply by rtivital** [reply]
For non controlled way, use HoverCard
